### PR TITLE
[Documents] Create configuration for documents #87:

### PR DESCRIPTION
**Pull request by codrin23**
#87 
- Added new configuration entity;
- fixed integration tests;
- added default configuration entity in service;
- marked with TODO comment the controller API method which should be modified.

### CODE REVIEW

File: src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.code4ro.legalconsultation.model.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class DocumentConfigutaionDto extends BaseEntityDto {
```
Comment (georgebejan) [inline src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java]: `DocumentConfigurationDto` small type

File: src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    private Boolean isOpenForCommenting;
```
Comment (georgebejan) [inline src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java]: there is no need for the `is` prefix - that would be included in the getter 
field name should be `openForCommenting`

File: src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    private Boolean isOpenForVotingComments;
```
Comment (georgebejan) [inline src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java]: same

File: src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import javax.persistence.*;
+
+@Entity
+@Table(name = "document_configuration")
+@Getter
+@Setter
+public class DocumentConfiguration extends BaseEntity {
+
+    @Column(name = "is_open_for_commenting")
+    private Boolean isOpenForCommenting;
```
Comment (georgebejan) [inline src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java]: same as in DTO

File: src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+    @Column(name = "is_open_for_voting_comments")
+    private Boolean isOpenForVotingComments;
```
Comment (georgebejan) [inline src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java]: same as in DTO

File: src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+    public DocumentConfiguration() {
+    }
+
+    public DocumentConfiguration(Boolean isOpenForCommenting, Boolean isOpenForVotingComments) {
```
Comment (georgebejan) [inline src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java]: you could use `@AllArgsConstructor` and `@NoArgsConstructor` from lombok

File: src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java (hunk 1 above)
Comment (codrin23) [reply src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java]: Done

File: src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java (hunk 2 above)
Comment (codrin23) [reply src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java]: Done

File: src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java (hunk 3 above)
Comment (codrin23) [reply src/main/java/com/code4ro/legalconsultation/model/dto/DocumentConfigutaionDto.java]: Done

File: src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java (hunk 4 above)
Comment (codrin23) [reply src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java]: Done

File: src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java (hunk 5 above)
Comment (codrin23) [reply src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java]: Done

File: src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java (hunk 6 above)
Comment (codrin23) [reply src/main/java/com/code4ro/legalconsultation/model/persistence/DocumentConfiguration.java]: Done

**Pull request merged**